### PR TITLE
feat(install): Kubernetes provider instructions

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: setup/providers/kubernetes.md [hunk 1]
```diff
@@ -6,3 +6,68 @@ sidebar:
...
 {% include toc %}
+
+In Kubernetes, an [__Account__](/setup/providers/overview#accounts) maps to a
+credential able to authenticate against your desired Kubernetes Cluster, as 
+well as a set of [Docker Registry](/setup/providers/docker-registry) accounts 
+to be used as a source of images.
+
+## Prerequisites
+
+Both the Kubernetes credentials and Docker accounts must exist before Halyard
```
Comment (duftler) [inline setup/providers/kubernetes.md]: Shouldn't this be 'Docker Registry accounts' as well?

File: setup/providers/overview.md [hunk 2]
```diff
@@ -7,28 +7,49 @@ sidebar:
...
 ## Supported Providers
 
 These are the cloud providers currently supported by Spinnaker:
 
 * [App Engine](/setup/providers/appengine)
 * [Amazon Web Services](/setup/providers/aws)
 * [Azure](/setup/providers/azure)
-* [Docker v2 Registry](/setup/providers/docker-registry) __Note:__ This is
-  only acts as a source of images, and does not include support for Docker Swarm.
+* [Docker v2 Registry](/setup/providers/docker-registry) (__Note:__ This is
```
Comment (duftler) [inline setup/providers/overview.md]: s/This is only acts/This only acts

**Review by duftler — APPROVED** [review]
LGTM after addressing comments.

File: setup/providers/kubernetes.md (hunk 1 above)
Comment (lwander) [reply setup/providers/kubernetes.md]: Yes, good call

**Pull request merged**